estate/models/estate_property.py

```diff
@@ -0,0 +1,117 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import float_compare
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Property'
+
+    name = fields.Char('Title', required=True)
+    description = fields.Text('Description')
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date(
+        'Available From',
+        copy=False,
+        default=fields.Date.add(fields.Date.today(), months=+3),
+    )
+    expected_price = fields.Float('Expected Price', required=True)
+    selling_price = fields.Float('Selling Price', readonly=True, copy=False)
+    bedrooms = fields.Integer('Bedrooms', default=2)
+    living_area = fields.Integer('Living Area (sqm)')
+    facades = fields.Integer('Facades')
+    garage = fields.Boolean('Garage')
+    garden = fields.Boolean('Garden')
+    garden_area = fields.Integer('Garden Area (sqm)')
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[
+            ('north', 'North'),
+            ('south', 'South'),
+            ('east', 'East'),
+            ('west', 'West'),
+        ],
+        default='north',
+    )
+    active = fields.Boolean('Active', default=True)
+    state = fields.Selection(
+        string='State',
+        selection=[
+            ('new', 'New'),
+            ('offer_received', 'Offer Received'),
+            ('offer_accepted', 'Offer Accepted'),
+            ('sold', 'Sold'),
+            ('canceled', 'Canceled'),
+        ],
+        required=True,
+        copy=False,
+        default='new',
+    )
+    property_type_id = fields.Many2one('estate.property.type', string='Property Types')
+    property_tag_ids = fields.Many2many('estate.property.tag', string='Property Tags')
+    buyer = fields.Many2one('res.partner', string='Buyer', copy=False)
+    salesperson = fields.Many2one(
```

Comment:
```suggestion
    buyer_id = fields.Many2one('res.partner', string='Buyer', copy=False)
    salesperson_id = fields.Many2one(
```
the naming convention for Many2one fields always should have `_id`